docs/css/extra.css

```diff
@@ -0,0 +1,14 @@
+pre code {
+  white-space: pre;
+  word-wrap: normal;
+  display: block;
+  padding: 12px;
+  font-size: 14px;
+}
+
+code {
```

Comment:
Here's an example of how we can override some stuff in the themes.

I thought the code blocks were too small